src/uu/sort/src/sort.rs

```diff
@@ -26,10 +27,25 @@ use std::path::Path;
 use uucore::fs::is_stdin_interactive; // for Iterator::dedup()
 
 static NAME: &str = "sort";
+static ABOUT: &str = "sort lines of text files";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
-const DECIMAL_PT: char = '.';
-const THOUSANDS_SEP: char = ',';
+static OPT_NUMERIC_SORT: &str = "numeric-sort";
+static OPT_HUMAN_NUMERIC_SORT: &str = "human-numeric-sort";
+static OPT_MONTH_SORT: &str = "month-sort";
+static OPT_VERSION_SORT: &str = "version-sort";
+static OPT_OUTPUT: &str = "output";
+static OPT_FILES: &str = "files";
+static OPT_MERGE: &str = "merge";
+static OPT_REVERSE: &str = "reverse";
+static OPT_STABLE: &str = "stable";
+static OPT_UNIQUE: &str = "unique";
+static OPT_CHECK: &str = "check";
+static OPT_DICTIONARY_ORDER: &str = "dictionary-order";
+static OPT_IGNORE_CASE: &str = "ignore-case";
```

Comment:
```suggestion
static OPT_HUMAN_NUMERIC_SORT: &str = "human-numeric-sort";
static OPT_MONTH_SORT: &str = "month-sort";
static OPT_NUMERIC_SORT: &str = "numeric-sort";
static OPT_VERSION_SORT: &str = "version-sort";

static OPT_DICTIONARY_ORDER: &str = "dictionary-order";
static OPT_MERGE: &str = "merge";
static OPT_CHECK: &str = "check";
static OPT_IGNORE_CASE: &str = "ignore-case";
static OPT_OUTPUT: &str = "output";
static OPT_REVERSE: &str = "reverse";
static OPT_STABLE: &str = "stable";
static OPT_UNIQUE: &str = "unique";

static ARG_FILES: &str = "files";
```

or just straight alphabetical for the `OPT_...` names if you think that's better?